service_capacity_modeling/models/org/netflix/ddb.py

```diff
@@ -0,0 +1,584 @@
+import math
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from pydantic import BaseModel
+from pydantic import Field
+
+from service_capacity_modeling.interface import AccessConsistency, certain_float
+from service_capacity_modeling.interface import AccessPattern
+from service_capacity_modeling.interface import CapacityDesires
+from service_capacity_modeling.interface import CapacityPlan
+from service_capacity_modeling.interface import CapacityRequirement
+from service_capacity_modeling.interface import certain_int
+from service_capacity_modeling.interface import Clusters
+from service_capacity_modeling.interface import Consistency
+from service_capacity_modeling.interface import DataShape
+from service_capacity_modeling.interface import Drive
+from service_capacity_modeling.interface import FixedInterval
+from service_capacity_modeling.interface import GlobalConsistency
+from service_capacity_modeling.interface import Instance
+from service_capacity_modeling.interface import Interval
+from service_capacity_modeling.interface import QueryPattern
+from service_capacity_modeling.interface import RegionContext
+from service_capacity_modeling.interface import Requirements
+from service_capacity_modeling.interface import Service
+from service_capacity_modeling.interface import ServiceCapacity
+from service_capacity_modeling.models import CapacityModel
+
+
+class NflxDynamoDBArguments(BaseModel):
+    number_of_regions: int = Field(
```

Comment:
This should be available in the RegionContext in structured arguments, no need for a model specific argument for this.

Reply:
This is not always available in the RegionContext, specifically if the approach is to plan for one region and then externally multiply with number of regions.

Reply:
It should always be accurate, if it is not that's a bug in the caller of this library. Even when planning in a single and extrapolating the AG endpoint passes the correct num_regions to the top level `plan` method  which fills in [num_regions](https://github.com/Netflix-Skunkworks/service-capacity-modeling/blob/1c408dee7c2b3b4377e60f5137a86b6e8f347af7/service_capacity_modeling/capacity_planner.py#L455). The only thing that plan in a single region should do is use pricing in only one region.

Reply:
Lets fix the caller then